tests/it/insert.rs

```diff
@@ -99,3 +99,28 @@ async fn overrides_client_options() {
     let rows = fetch_simple_rows(&client, table_name).await;
     assert_eq!(rows, vec!(row))
 }
+
+#[tokio::test]
+async fn empty_insert() {
+    // https://github.com/ClickHouse/clickhouse-rs/issues/137
+
+    let table_name = "insert_empty";
+    let query_id = uuid::Uuid::new_v4().to_string();
+    let (setting_name, setting_value, override_value) = ("async_insert", "0", "1");
```

Comment:
query_id and settings are not needed here (irrelevant to an empty insert test), as well as `flush_query_log` call